charts/studio/values.yaml

```diff
@@ -551,10 +566,96 @@ studioWorker:
     # runAsNonRoot: true
     # runAsUser: 1000
 
+  # -- Worker node selector
+  nodeSelector: {}
+  # -- Worker tolerations
+  tolerations: []
+  # -- Worker affinity
+  affinity: {}
+
+# -- Studio DVCx Worker settings group
+studioDvcxWorker:
+
+  # -- DVCx worker image settings
+  image:
+    # -- DVCx worker image repository
+    repository: docker.iterative.ai/studio-backend
```

Comment:
This also needs to be changed when we will have separate images for worker @iterative/platform